src/applications/appeals/10182/components/ContactInformation.jsx

```diff
@@ -104,100 +74,98 @@ export const ContactInfoDescription = ({
             {missingInfo[0].startsWith('p') ? ' A ' : ' An '}
             {list} {plural ? 'are' : 'is'} required for this application.
           </p>
-          {(hasUpdated || submitted) && (
+          {submitted && (
             <div className="vads-u-margin-top--1p5" role="alert">
               <va-alert status="error" background-only>
                 <div className="vads-u-font-size--base">
-                  We still don’t have your {list}. Please{' '}
-                  {profileLink(`add your ${list} in your profile.`)}
+                  We still don’t have your {list}. Please edit and update the
+                  field.
                 </div>
               </va-alert>
             </div>
           )}
           <div className="vads-u-margin-top--1p5" role="alert">
             <va-alert status="warning" background-only>
               <div className="vads-u-font-size--base">
-                Your {list} {plural ? 'are' : 'is'} missing. Please{' '}
-                {profileLink(`add ${plural ? 'them' : 'it'} in your profile`)}{' '}
-                and return to this page. When you’re done, please click the
-                button to confirm your {list} {plural ? 'have' : 'has'} been
-                updated.
+                Your {list} {plural ? 'are' : 'is'} missing. Please edit and
+                update the {plural ? 'fields' : 'field'}.
               </div>
-              <button
-                type="button"
-                className="vads-u-width--auto"
-                onClick={event => {
-                  event.preventDefault();
-                  setHasUpdated(true);
-                  refreshProfile(true);
-                }}
-              >
-                My contact details have been updated
-              </button>
             </va-alert>
           </div>
         </>
       )}
-      <div className="blue-bar-block">
-        <h3 className="vads-u-font-size--h4">Phone &amp; email</h3>
-        <p>
-          <strong>{phoneType} phone</strong>:{' '}
-          {phone?.phoneNumber && (
-            <Telephone
-              contact={`${phone?.areaCode || ''}${phone?.phoneNumber || ''}`}
-              extension={phone?.extension}
-              notClickable
-            />
-          )}
-        </p>
-        <p>
-          <strong>Email address</strong>: {email?.emailAddress || ''}
-        </p>
-        <h3 className="vads-u-font-size--h4">Mailing address</h3>
-        <p>
-          {addBrAfter(street)}
-          {addBrAfter(cityStateZip)}
-          {country}
-        </p>
+      <div className="blue-bar-block vads-u-margin-top--4">
+        <div
+          className="va-profile-wrapper"
+          onSubmit={event => event.stopPropagation()}
```

Comment:
This prevents this nested form submit event from passing to the outer form and causing a page advance.